sclack/components.py

```diff
@@ -564,13 +564,26 @@ def keypress(self, size, key):
         return super(MessagePrompt, self).keypress(size, key)
 
 class Profile(urwid.Text):
-    def __init__(self, name, is_online=False):
-        if is_online:
+    def __init__(self, name, is_online=False, is_snoozed=False):
+        self.name = name
+        self.is_online = is_online
+        self.is_snoozed = is_snoozed
+        super(Profile, self).__init__(self.body)
+
+    @property
+    def body(self):
+        if self.is_snoozed:
+            presence_icon = ('presence_snooze', ' {} '.format(get_icon('snooze')))
```

Comment:
Could you please change `presence_snooze` to `presence_active`, It is actually the color of the icon
So the icon becomes
![snooze](https://i.imgur.com/at0XuwE.png)

Reply:
Aaaah, got it, thanks. I'll do.

Should this 'snoozed' string be there too?

Reply:
I think yes, good to have, because not everyone use the icon so the text should be helpful